README.rst

```diff
@@ -57,63 +74,40 @@ Compatibility Notes
 
 **NOTE**: The Maple version does not support Django 2.2 but it does support Django 3.2 as of eox-core 7.0.
 
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Lilac**
-
-.. code-block:: yaml
+The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied in order to be used for previous releases.
 
-   EOX_CORE_USERS_BACKEND: "eox_core.edxapp_wrapper.backends.users_l_v1"
-   EOX_CORE_PRE_ENROLLMENT_BACKEND: "eox_core.edxapp_wrapper.backends.pre_enrollment_l_v1"
-   EOX_CORE_ENROLLMENT_BACKEND: "eox_core.edxapp_wrapper.backends.enrollment_l_v1"
+If the release you are looking for is not listed, the actual configuration is sufficient or it is incompatible with the current eox-core version.
```

Comment:
I thought that if wasn't not listed, it's not supported. In what case does the other scenario happen?

Reply:
Same explanation https://github.com/eduNEXT/eox-tenant/pull/210#discussion_r1650321069

Reply:
Mmm, I get it now that you explained it. Back when I left the comment, I was thinking about the ambiguity of "or the configuration is enough or is not supported." How do I know?

Still, this is a non-blocking comment.

Reply:
Understanding that I add this line below

>  If you are installing a previous version, please refer to the tag to verify the configuration that should be applied.

I can think of a better way to explain it if it is still vague. :)

Reply:
That sounds good!

Reply:
We could follow this suggestion as well: https://github.com/eduNEXT/eox-tenant/pull/210#discussion_r1654992895, it seems clearer.